src/app/products/products.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="products-container">
+  <nav class="sidebar">
+    <ul class="navbar">
+      {{#each categories.categories}}
+        <li class="nav-item">
+          <a class="nav-item-link item-{{@index}}" href="#">
+            <span>{{name}}</span>
+          </a>
+        </li>
+      {{/each}}
+    </ul>
+  </nav>
+  <nav class="topbar">
+    <select class="navbar selected">
+      {{#each categories.categories}}
+        <option class="nav-item">
+          <span class="nav-item-link">{{name}}</span>
+        </option>
+      {{/each}}
+    </select>
+  </nav>
+  <ul class="products-showcase">
+    {{#each products.allProducts}}
+    <li>
+      <h4 tabindex="0">{{name}}</h4>
+      <div class="product-box">
+        <figure class="product-photo">
+          <img tabindex="0" src="{{imageURL}}" alt="{{name}}">
+        </figure>
+        <p tabindex="0">{{description}}</p>
```

Comment:
why you suing tab index so much, and on P it doesn't make any sense

Reply:
Tab Index because want the user to read out the description of the products as well.